requirements.txt

```diff
@@ -19,7 +19,7 @@ decorator==4.4.2          # via jsonpath-rw, networkx
 glob2==0.7                # via packtivity, yadage
 graphviz==0.14.1          # via reana-workflow-engine-yadage (setup.py)
 idna==2.8                 # via jsonschema, requests
-jq==1.0.2                 # via packtivity, reana-workflow-engine-yadage (setup.py), yadage
+jq==0.1.7                 # via packtivity, reana-workflow-engine-yadage (setup.py), yadage
```

Comment:
Nice, the pinning solved the slowness for me :tada: 

![2020-09-22-174039_718x112_scrot](https://user-images.githubusercontent.com/517546/93905089-dc85c800-fcfa-11ea-92ee-b8681ff0de89.png)

We can merge and release and hunt for the ReadTheDocs build troubles separately afterwards...